hacktoberfest_challenges/model_license_other.md

```diff
@@ -0,0 +1,410 @@
+
```

Comment:
Please put the license in the documentation images dataset on the hub

Reply:
Yes, right :+1: